package.json

```diff
@@ -24,7 +24,8 @@
     "ramda": "^0.26.1",
     "react": "^16.10.2",
     "react-dom": "^16.10.2",
-    "styled-components": "^4.4.0"
+    "styled-components": "^4.4.0",
+    "rmwc": "^5.7"
```

Comment:
Let's install on per-component basis. It's also recommended in rmwc docs.

Reply:
Ok. I had it like that in the beginning but then I remember we said that webpack will selectively include what is required only. 
I'll put it back on a separate per component basis...